dictionaries/software-terms/src/software-tools.txt

```diff
@@ -4,6 +4,7 @@ airbrake
 AirWatch
 AKS
 Alibaba
+Appointedd
```

Comment:
In most cases, company names are sufficient. Are these actual tools or just websites / services?

Maybe it is time to make a software-services list.

Reply:
They are just services, a software services list is a great idea! I'll adjust this PR